mne/connectivity/envelope.py

```diff
@@ -82,6 +84,11 @@ def envelope_correlation(data, combine='mean', verbose=None):
         if epoch_data.dtype in (np.float32, np.float64):
             n_fft = next_fast_len(n_times)
             epoch_data = hilbert(epoch_data, N=n_fft, axis=-1)[..., :n_times]
+
+        if orthogonalize is False:
+            corrs.append(np.corrcoef(np.abs(epoch_data)))
+            continue
```

Comment:
Rather than just `if ...: ...; continue` I'd prefer to factor out the orthogonalization below. It makes it clearer when editing the code in the future what the orthogonalization is doing, and how close a standard correlation coefficient is to it.

For example you can make use of `data_mag`, `data_mag_nomean`, and `data_mag_std` (where we are careful not to create `nan` values, unlike `corrcoef`), then do:
```
for li, label_data in enumerate(epoch_data):
    if orthogonalize is False:  # the new code
        label_data_orth = label_data
        label_data_orth_std = data_mag_std[li]
    else:  # the existing code
       label_data_orth = ...
       label_data_orth_std= ...
    # more existing code
    corr[li] = np.dot(label_data_orth, data_mag_nomean[li])
    corr[li] /= data_mag_std[li]
    corr[li] /= label_data_orth_std
if orthogonalize is not False:
    corr = np.abs(corr)
```